### PR TITLE
Removed deprecated methods in Message class, that were used only in u…

### PR DESCRIPTION
**Pull request by luk-kaminski**
…nit tests.

## Description
Removed deprecated methods in Message class, that were used only in unit tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by bernd — CHANGES_REQUESTED** [review]
Please document the removed methods in `UPGRADING.md`. This is a breaking change for 3rd-party plugins.

**Review by bernd — CHANGES_REQUESTED** [review]
And this change also needs a changelog entry.

**Comment by luk-kaminski** [comment]
> Please document the removed methods in `UPGRADING.md`. This is a breaking change for 3rd-party plugins.

Would it be enough to use the table in `Java API Changes` section of `UPGRADING.md` for that, or do you want to have a completely separate section for this change?

**Comment by bernd** [comment]
> > Please document the removed methods in `UPGRADING.md`. This is a breaking change for 3rd-party plugins.
> 
> Would it be enough to use the table in `Java API Changes` section of `UPGRADING.md` for that, or do you want to have a completely separate section for this change?

An addition to the "Java API Changes" section and the changelog entry about the removal should be enough. :+1:

File: UPGRADING.md [hunk 1]
```diff
@@ -38,9 +38,12 @@ Any existing Active Directory User Asset import configurations will be automatic
...
 
-| File/method                    | Description              |
-|--------------------------------|--------------------------|
-| `ExampleClass#exampleFunction` | TODO placeholder comment |
+| File/method                    | Description               |
+|--------------------------------|---------------------------|
+| `org.graylog2.plugin.Message#addStringFields` | Deprecated method removed |
+| `org.graylog2.plugin.Message#addLongFields` | Deprecated method removed |
+| `org.graylog2.plugin.Message#addDoubleFields` | Deprecated method removed |
+
```
Comment (bernd) [inline UPGRADING.md:46]: The `#getValidationErrors` method is missing in the list. (and in the changelog entry)

**Review by bernd — CHANGES_REQUESTED** [review]

File: UPGRADING.md (hunk 1 above)
Comment (luk-kaminski) [reply UPGRADING.md:47]: Thx for spotting it!

**Review by bernd — APPROVED** [review]

**Pull request merged**